.github/workflows/python-app.yml

```diff
@@ -18,6 +18,9 @@ jobs:
       - name: Lint with flake8
         run: |
           poetry run flake8 . --count --select=E9,F63,F7,F82 --max-complexity=10 --max-line-length=127 --show-source --statistics
+      - name: Measure complexity with Radon
+        run: |
+          cd soil-moisture-sensor && poetry run radon cc -a app.py
```

Comment:
Radon not designed for CI/CD usage, and should not exclusively run on app.py, consider using https://github.com/rubik/xenon xenon by the same person instead